services/graylog/scripts/configure.py

```
@@ -139,17 +145,96 @@ def configure_email_notifications(_session, _headers):
     _mail_notification_id = [
         noti
         for noti in _r.json()["notifications"]
-        if noti["title"] == "Graylog " + env.str("MACHINE_FQDN") + " mail notification"
+        if noti["title"] == "Graylog " + MACHINE_FQDN + " mail notification"
     ][0]["id"]
 
     return _mail_notification_id
 
 
+def configure_slack_notification_channel(_session, _hed) -> str:
```

Is there any test / way to verify that these notifications are configured properly and reach the final destination? 

(to avoid alerts that silently fail to reach slack)

---

I am not sure this can be done in a programmatic way in a satisfactory way. However, the graylog dashboards to the best of my knowledge indicate if alert channels cannot be reached. If you have a usggestion how to test this please let me know :--) i am open to it!